### ISSUE TITLE
Release Chrome version of this Extension

### ISSUE THREAD

**Issue by blackwind** [meh/smart-referer#94]
Now that this add-on has been converted to WebExtensions, is there any chance we'll see a Chrome version? I'm a Firefox user first and foremost, but we all end up having to use Chrome sometimes for one reason or another. It'd be great to have Smart Referer available for those occasions.

Are any Firefox-specific APIs currently being used? And if so, are they absolutely necessary, or could they be replaced with ones available in other browsers?

**Comment by alexander255** [comment]
Sorry for ignoring you a few days!
Chrome support very likely doable, but I just really don't care to-be-honest. If you (or anybody else) want to take a stab at this feel free and if the patches aren't too invasive I'll even merge them. I will not however do any testing and explicit maintanace for Chrome and I will also not add the extension to Chrome's marketplace – somebody else will need to do that (ideally asking me first!) and person better stick around for addressing Chrome-specific issues.

TL;DR: I don't give a damn, but if anybody wants to take a stab at it I won't stand in your way and I'd answer some questions about the code. If this is to happen as part of this repository, I want that person to stick around and do the maintanence and support as well as uploading new versions.

**Comment by ntninja** [comment]
This issue has been moved to GitLab: https://gitlab.com/smart-referer/smart-referer/issues/94
